pyam/utils.py

```diff
@@ -208,8 +208,29 @@ def style_df(df, style='heatmap'):
         return df.style.background_gradient(cmap=cm)
 
 
-def find_depth(data, s, level):
-    # determine function for finding depth level =, >=, <= |s
+def find_depth(data, s='', level=None):
+    """
+    return a list of bools asserting the depth indicated by `level`;
```

Comment:
this docstring is getting long (and a bit hard to comprehend). would you please go ahead and make it "official" with a `Params` and `Returns` section?

Reply:
done